airootfs/usr/share/vx-img/flash-image.sh

```diff
@@ -1,15 +1,16 @@
 #!/bin/bash
 
-# TODO use dmidecode to detect if we're on a Surface Go
-if sudo dmidecode | grep 'Surface Go'; then
+clear
```

Comment:
what are these for?

Reply:
it just makes the screen less cluttered, and helps the steps feel more discrete.